ee/hogai/eval/ci/max_tools/eval_create_experiment_tool.py

```diff
@@ -65,103 +65,179 @@ def _run_eval_sync(self, output: dict, expected: dict, **kwargs):
         return Score(name=self._name(), score=total_score, metadata=metadata)
 
 
-@pytest.mark.django_db
-async def eval_create_experiment_basic(pytestconfig, demo_org_team_user):
-    """Test basic experiment creation."""
-    _, team, user = demo_org_team_user
-
-    conversation = await Conversation.objects.acreate(team=team, user=user)
-
-    async def task_create_experiment(args: dict):
-        # Create feature flag first (required by the tool)
-        await FeatureFlag.objects.acreate(
-            team=team,
-            created_by=user,
-            key=args["feature_flag_key"],
-            name=f"Flag for {args['name']}",
-            filters={
-                "groups": [{"properties": [], "rollout_percentage": 100}],
-                "multivariate": {
-                    "variants": [
-                        {"key": "control", "name": "Control", "rollout_percentage": 50},
-                        {"key": "test", "name": "Test", "rollout_percentage": 50},
-                    ]
+class TestCreateExperimentTool:
+    @pytest.mark.django_db
+    async def eval_create_experiment_basic(self, pytestconfig, demo_org_team_user):
```

Comment:
This is not how our evals are structured still! In fact in terms of the output nothing has changed:

<img width="748" height="601" alt="Image" src="https://github.com/user-attachments/assets/292cef91-0146-478e-97dd-db312a66d343" />

All of these should be part of one `tool_create_experiment` eval suite (i.e. one `MaxPublicEval()`), as they share the same scorer.